Description: Upstream changes introduced in version 1.7.4-2.3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 encfs (1.7.4-2.3) unstable; urgency=low
 .
   * Non-maintainer upload.
   * debian/control: Switch to unversioned Boost build-dependencies.
     (Closes: #630436).
 .
 The person named in the Author field signed this changelog entry.
Author: Steve M. Robbins <smr@debian.org>
Bug-Debian: http://bugs.debian.org/630436

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ encfs-1.7.4/config.log
@@ -0,0 +1,1097 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by configure, which was
+generated by GNU Autoconf 2.68.  Invocation command line was
+
+  $ ./configure --build=x86_64-linux-gnu --prefix=/usr --mandir=${prefix}/share/man --infodir=${prefix}/share/info
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = riemann
+uname -m = x86_64
+uname -r = 2.6.39-2-amd64
+uname -s = Linux
+uname -v = #1 SMP Wed Jun 8 11:01:04 UTC 2011
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2571: checking for a BSD-compatible install
+configure:2639: result: /usr/bin/install -c
+configure:2650: checking whether build environment is sane
+configure:2700: result: yes
+configure:2841: checking for a thread-safe mkdir -p
+configure:2880: result: /bin/mkdir -p
+configure:2893: checking for gawk
+configure:2909: found /usr/bin/gawk
+configure:2920: result: gawk
+configure:2931: checking whether make sets $(MAKE)
+configure:2953: result: yes
+configure:3037: checking build system type
+configure:3051: result: x86_64-pc-linux-gnu
+configure:3071: checking host system type
+configure:3084: result: x86_64-pc-linux-gnu
+configure:3184: checking for g++
+configure:3200: found /usr/bin/g++
+configure:3211: result: g++
+configure:3238: checking for C++ compiler version
+configure:3247: g++ --version >&5
+g++ (Debian 4.6.1-4) 4.6.1
+Copyright (C) 2011 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3258: $? = 0
+configure:3247: g++ -v >&5
+Using built-in specs.
+COLLECT_GCC=g++
+COLLECT_LTO_WRAPPER=/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/lto-wrapper
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.6.1-4' --with-bugurl=file:///usr/share/doc/gcc-4.6/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++,go --prefix=/usr --program-suffix=-4.6 --enable-shared --enable-multiarch --with-multiarch-defaults=x86_64-linux-gnu --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib/x86_64-linux-gnu --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.6 --libdir=/usr/lib/x86_64-linux-gnu --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-plugin --enable-objc-gc --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.6.1 (Debian 4.6.1-4) 
+configure:3258: $? = 0
+configure:3247: g++ -V >&5
+g++: error: unrecognized option '-V'
+g++: fatal error: no input files
+compilation terminated.
+configure:3258: $? = 1
+configure:3247: g++ -qversion >&5
+g++: error: unrecognized option '-qversion'
+g++: fatal error: no input files
+compilation terminated.
+configure:3258: $? = 1
+configure:3278: checking whether the C++ compiler works
+configure:3300: g++ -g -O2   conftest.cpp  >&5
+configure:3304: $? = 0
+configure:3352: result: yes
+configure:3355: checking for C++ compiler default output file name
+configure:3357: result: a.out
+configure:3363: checking for suffix of executables
+configure:3370: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3374: $? = 0
+configure:3396: result: 
+configure:3418: checking whether we are cross compiling
+configure:3426: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3430: $? = 0
+configure:3437: ./conftest
+configure:3441: $? = 0
+configure:3456: result: no
+configure:3461: checking for suffix of object files
+configure:3483: g++ -c -g -O2  conftest.cpp >&5
+configure:3487: $? = 0
+configure:3508: result: o
+configure:3512: checking whether we are using the GNU C++ compiler
+configure:3531: g++ -c -g -O2  conftest.cpp >&5
+configure:3531: $? = 0
+configure:3540: result: yes
+configure:3549: checking whether g++ accepts -g
+configure:3569: g++ -c -g  conftest.cpp >&5
+configure:3569: $? = 0
+configure:3610: result: yes
+configure:3644: checking for style of include used by make
+configure:3672: result: GNU
+configure:3697: checking dependency style of g++
+configure:3807: result: gcc3
+configure:3827: checking whether NLS is requested
+configure:3836: result: yes
+configure:3877: checking for msgfmt
+configure: trying /usr/bin/msgfmt...
+0 translated messages.
+configure:3909: result: /usr/bin/msgfmt
+configure:3918: checking for gmsgfmt
+configure:3949: result: /usr/bin/msgfmt
+configure:4000: checking for xgettext
+configure: trying /usr/bin/xgettext...
+/usr/bin/xgettext: warning: file `/dev/null' extension `' is unknown; will try C
+configure:4032: result: /usr/bin/xgettext
+configure:4078: checking for msgmerge
+configure: trying /usr/bin/msgmerge...
+configure:4109: result: /usr/bin/msgmerge
+configure:4190: checking for gcc
+configure:4206: found /usr/bin/gcc
+configure:4217: result: gcc
+configure:4446: checking for C compiler version
+configure:4455: gcc --version >&5
+gcc (Debian 4.6.1-4) 4.6.1
+Copyright (C) 2011 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:4466: $? = 0
+configure:4455: gcc -v >&5
+Using built-in specs.
+COLLECT_GCC=gcc
+COLLECT_LTO_WRAPPER=/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/lto-wrapper
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.6.1-4' --with-bugurl=file:///usr/share/doc/gcc-4.6/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++,go --prefix=/usr --program-suffix=-4.6 --enable-shared --enable-multiarch --with-multiarch-defaults=x86_64-linux-gnu --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib/x86_64-linux-gnu --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.6 --libdir=/usr/lib/x86_64-linux-gnu --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-plugin --enable-objc-gc --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.6.1 (Debian 4.6.1-4) 
+configure:4466: $? = 0
+configure:4455: gcc -V >&5
+gcc: error: unrecognized option '-V'
+gcc: fatal error: no input files
+compilation terminated.
+configure:4466: $? = 1
+configure:4455: gcc -qversion >&5
+gcc: error: unrecognized option '-qversion'
+gcc: fatal error: no input files
+compilation terminated.
+configure:4466: $? = 1
+configure:4470: checking whether we are using the GNU C compiler
+configure:4489: gcc -c -Wall -g -O2  conftest.c >&5
+configure:4489: $? = 0
+configure:4498: result: yes
+configure:4507: checking whether gcc accepts -g
+configure:4527: gcc -c -g  conftest.c >&5
+configure:4527: $? = 0
+configure:4568: result: yes
+configure:4585: checking for gcc option to accept ISO C89
+configure:4649: gcc  -c -Wall -g -O2  conftest.c >&5
+configure:4649: $? = 0
+configure:4662: result: none needed
+configure:4684: checking dependency style of gcc
+configure:4794: result: gcc3
+configure:4833: checking for ld used by GCC
+configure:4897: result: /usr/bin/ld
+configure:4904: checking if the linker (/usr/bin/ld) is GNU ld
+configure:4917: result: yes
+configure:4924: checking for shared library run path origin
+configure:4937: result: done
+configure:5439: checking for CFPreferencesCopyAppValue
+configure:5457: g++ -o conftest -g -O2   conftest.cpp  -Wl,-framework -Wl,CoreFoundation >&5
+conftest.cpp:11:42: fatal error: CoreFoundation/CFPreferences.h: No such file or directory
+compilation terminated.
+configure:5457: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| /* end confdefs.h.  */
+| #include <CoreFoundation/CFPreferences.h>
+| int
+| main ()
+| {
+| CFPreferencesCopyAppValue(NULL, NULL)
+|   ;
+|   return 0;
+| }
+configure:5466: result: no
+configure:5473: checking for CFLocaleCopyCurrent
+configure:5491: g++ -o conftest -g -O2   conftest.cpp  -Wl,-framework -Wl,CoreFoundation >&5
+conftest.cpp:11:37: fatal error: CoreFoundation/CFLocale.h: No such file or directory
+compilation terminated.
+configure:5491: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| /* end confdefs.h.  */
+| #include <CoreFoundation/CFLocale.h>
+| int
+| main ()
+| {
+| CFLocaleCopyCurrent();
+|   ;
+|   return 0;
+| }
+configure:5500: result: no
+configure:5549: checking for GNU gettext in libc
+configure:5569: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:5569: $? = 0
+configure:5578: result: yes
+configure:6324: checking whether to use NLS
+configure:6326: result: yes
+configure:6329: checking where the gettext function comes from
+configure:6340: result: libc
+configure:6523: checking how to print strings
+configure:6550: result: printf
+configure:6571: checking for a sed that does not truncate output
+configure:6635: result: /bin/sed
+configure:6653: checking for grep that handles long lines and -e
+configure:6711: result: /bin/grep
+configure:6716: checking for egrep
+configure:6778: result: /bin/grep -E
+configure:6783: checking for fgrep
+configure:6845: result: /bin/grep -F
+configure:6880: checking for ld used by gcc
+configure:6947: result: /usr/bin/ld
+configure:6954: checking if the linker (/usr/bin/ld) is GNU ld
+configure:6969: result: yes
+configure:6981: checking for BSD- or MS-compatible name lister (nm)
+configure:7030: result: /usr/bin/nm -B
+configure:7160: checking the name lister (/usr/bin/nm -B) interface
+configure:7167: g++ -c -g -O2  conftest.cpp >&5
+configure:7170: /usr/bin/nm -B "conftest.o"
+configure:7173: output
+0000000000000000 B some_variable
+configure:7180: result: BSD nm
+configure:7183: checking whether ln -s works
+configure:7187: result: yes
+configure:7195: checking the maximum length of command line arguments
+configure:7320: result: 3458764513820540925
+configure:7337: checking whether the shell understands some XSI constructs
+configure:7347: result: yes
+configure:7351: checking whether the shell understands "+="
+configure:7357: result: yes
+configure:7392: checking for /usr/bin/ld option to reload object files
+configure:7399: result: -r
+configure:7468: checking for objdump
+configure:7484: found /usr/bin/objdump
+configure:7495: result: objdump
+configure:7527: checking how to recognize dependent libraries
+configure:7729: result: pass_all
+configure:7789: checking for ar
+configure:7805: found /usr/bin/ar
+configure:7816: result: ar
+configure:7894: checking for strip
+configure:7910: found /usr/bin/strip
+configure:7921: result: strip
+configure:7993: checking for ranlib
+configure:8009: found /usr/bin/ranlib
+configure:8020: result: ranlib
+configure:8122: checking command to parse /usr/bin/nm -B output from gcc object
+configure:8240: g++ -c -g -O2  conftest.cpp >&5
+configure:8243: $? = 0
+configure:8247: /usr/bin/nm -B conftest.o \| sed -n -e 's/^.*[ ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[ ][ ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' \> conftest.nm
+configure:8250: $? = 0
+configure:8304: g++ -o conftest -g -O2   conftest.cpp conftstm.o >&5
+configure:8307: $? = 0
+configure:8345: result: ok
+configure:8441: g++ -c -g -O2  conftest.cpp >&5
+configure:8444: $? = 0
+configure:9186: checking how to run the C preprocessor
+configure:9217: gcc -E  conftest.c
+configure:9217: $? = 0
+configure:9231: gcc -E  conftest.c
+conftest.c:14:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:9231: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:9256: result: gcc -E
+configure:9276: gcc -E  conftest.c
+configure:9276: $? = 0
+configure:9290: gcc -E  conftest.c
+conftest.c:14:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:9290: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:9319: checking for ANSI C header files
+configure:9339: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9339: $? = 0
+configure:9412: gcc -o conftest -Wall -g -O2   conftest.c  >&5
+configure:9412: $? = 0
+configure:9412: ./conftest
+configure:9412: $? = 0
+configure:9423: result: yes
+configure:9436: checking for sys/types.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for sys/stat.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for stdlib.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for string.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for memory.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for strings.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for inttypes.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for stdint.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9436: checking for unistd.h
+configure:9436: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9436: $? = 0
+configure:9436: result: yes
+configure:9450: checking for dlfcn.h
+configure:9450: gcc -c -Wall -g -O2  conftest.c >&5
+configure:9450: $? = 0
+configure:9450: result: yes
+configure:9577: checking for objdir
+configure:9592: result: .libs
+configure:9863: checking if gcc supports -fno-rtti -fno-exceptions
+configure:9881: gcc -c -Wall -g -O2  -fno-rtti -fno-exceptions conftest.c >&5
+cc1: warning: command line option '-fno-rtti' is valid for C++/ObjC++ but not for C [enabled by default]
+configure:9885: $? = 0
+configure:9898: result: no
+configure:9918: checking for gcc option to produce PIC
+configure:10203: result: -fPIC -DPIC
+configure:10215: checking if gcc PIC flag -fPIC -DPIC works
+configure:10233: gcc -c -Wall -g -O2  -fPIC -DPIC -DPIC conftest.c >&5
+configure:10237: $? = 0
+configure:10250: result: yes
+configure:10274: checking if gcc static flag -static works
+configure:10302: result: yes
+configure:10317: checking if gcc supports -c -o file.o
+configure:10338: gcc -c -Wall -g -O2  -o out/conftest2.o conftest.c >&5
+configure:10342: $? = 0
+configure:10364: result: yes
+configure:10372: checking if gcc supports -c -o file.o
+configure:10419: result: yes
+configure:10452: checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:11527: result: yes
+configure:11564: checking whether -lc should be explicitly linked in
+configure:11572: gcc -c -Wall -g -O2  conftest.c >&5
+configure:11575: $? = 0
+configure:11590: gcc -shared conftest.o  -v -Wl,-soname -Wl,conftest -o conftest 2\>\&1 \| /bin/grep  -lc  \>/dev/null 2\>\&1
+configure:11593: $? = 0
+configure:11607: result: no
+configure:11772: checking dynamic linker characteristics
+configure:12227: gcc -o conftest -Wall -g -O2   -Wl,-rpath -Wl,/foo conftest.c  >&5
+configure:12227: $? = 0
+configure:12449: result: GNU/Linux ld.so
+configure:12556: checking how to hardcode library paths into programs
+configure:12581: result: immediate
+configure:13121: checking whether stripping libraries is possible
+configure:13126: result: yes
+configure:13161: checking if libtool supports shared libraries
+configure:13163: result: yes
+configure:13166: checking whether to build shared libraries
+configure:13187: result: yes
+configure:13190: checking whether to build static libraries
+configure:13194: result: no
+configure:13217: checking how to run the C++ preprocessor
+configure:13244: g++ -E  conftest.cpp
+configure:13244: $? = 0
+configure:13258: g++ -E  conftest.cpp
+conftest.cpp:26:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:13258: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:13283: result: g++ -E
+configure:13303: g++ -E  conftest.cpp
+configure:13303: $? = 0
+configure:13317: g++ -E  conftest.cpp
+conftest.cpp:26:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:13317: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:13485: checking for ld used by g++
+configure:13552: result: /usr/bin/ld -m elf_x86_64
+configure:13559: checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld
+configure:13574: result: yes
+configure:13629: checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:14561: result: yes
+configure:14589: g++ -c -g -O2  conftest.cpp >&5
+configure:14592: $? = 0
+configure:14774: checking for g++ option to produce PIC
+configure:15096: result: -fPIC -DPIC
+configure:15105: checking if g++ PIC flag -fPIC -DPIC works
+configure:15123: g++ -c -g -O2  -fPIC -DPIC -DPIC conftest.cpp >&5
+configure:15127: $? = 0
+configure:15140: result: yes
+configure:15161: checking if g++ static flag -static works
+configure:15189: result: yes
+configure:15201: checking if g++ supports -c -o file.o
+configure:15222: g++ -c -g -O2  -o out/conftest2.o conftest.cpp >&5
+configure:15226: $? = 0
+configure:15248: result: yes
+configure:15253: checking if g++ supports -c -o file.o
+configure:15300: result: yes
+configure:15330: checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
+configure:15358: result: yes
+configure:15501: checking dynamic linker characteristics
+configure:16112: result: GNU/Linux ld.so
+configure:16165: checking how to hardcode library paths into programs
+configure:16190: result: immediate
+configure:16413: checking for the pthreads library -lpthreads
+configure:16446: gcc -o conftest -Wall -g -O2    conftest.c -lpthreads  >&5
+conftest.c: In function 'main':
+conftest.c:31:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 3) [-Wnonnull]
+conftest.c:30:27: warning: 'th' is used uninitialized in this function [-Wuninitialized]
+/usr/bin/ld: cannot find -lpthreads
+collect2: ld returned 1 exit status
+configure:16446: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <pthread.h>
+| int
+| main ()
+| {
+| pthread_t th; pthread_join(th, 0);
+|                      pthread_attr_init(0); pthread_cleanup_push(0, 0);
+|                      pthread_create(0,0,0,0); pthread_cleanup_pop(0);
+|   ;
+|   return 0;
+| }
+configure:16455: result: no
+configure:16358: checking whether pthreads work without any flags
+configure:16446: gcc -o conftest -Wall -g -O2    conftest.c   >&5
+conftest.c: In function 'main':
+conftest.c:31:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 3) [-Wnonnull]
+conftest.c:30:27: warning: 'th' is used uninitialized in this function [-Wuninitialized]
+/tmp/ccffsUAZ.o: In function `main':
+/home/steve/Packages/NMU/encfs-1.7.4/conftest.c:30: undefined reference to `pthread_join'
+/home/steve/Packages/NMU/encfs-1.7.4/conftest.c:31: undefined reference to `__pthread_register_cancel'
+/home/steve/Packages/NMU/encfs-1.7.4/conftest.c:32: undefined reference to `pthread_create'
+/home/steve/Packages/NMU/encfs-1.7.4/conftest.c:32: undefined reference to `__pthread_unregister_cancel'
+collect2: ld returned 1 exit status
+configure:16446: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <pthread.h>
+| int
+| main ()
+| {
+| pthread_t th; pthread_join(th, 0);
+|                      pthread_attr_init(0); pthread_cleanup_push(0, 0);
+|                      pthread_create(0,0,0,0); pthread_cleanup_pop(0);
+|   ;
+|   return 0;
+| }
+configure:16455: result: no
+configure:16363: checking whether pthreads work with -Kthread
+configure:16446: gcc -o conftest -Wall -g -O2 -Kthread   conftest.c   >&5
+gcc: error: unrecognized option '-Kthread'
+configure:16446: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <pthread.h>
+| int
+| main ()
+| {
+| pthread_t th; pthread_join(th, 0);
+|                      pthread_attr_init(0); pthread_cleanup_push(0, 0);
+|                      pthread_create(0,0,0,0); pthread_cleanup_pop(0);
+|   ;
+|   return 0;
+| }
+configure:16455: result: no
+configure:16363: checking whether pthreads work with -kthread
+configure:16446: gcc -o conftest -Wall -g -O2 -kthread   conftest.c   >&5
+gcc: error: unrecognized option '-kthread'
+configure:16446: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <pthread.h>
+| int
+| main ()
+| {
+| pthread_t th; pthread_join(th, 0);
+|                      pthread_attr_init(0); pthread_cleanup_push(0, 0);
+|                      pthread_create(0,0,0,0); pthread_cleanup_pop(0);
+|   ;
+|   return 0;
+| }
+configure:16455: result: no
+configure:16413: checking for the pthreads library -llthread
+configure:16446: gcc -o conftest -Wall -g -O2    conftest.c -llthread  >&5
+conftest.c: In function 'main':
+conftest.c:31:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 3) [-Wnonnull]
+conftest.c:30:27: warning: 'th' is used uninitialized in this function [-Wuninitialized]
+/usr/bin/ld: cannot find -llthread
+collect2: ld returned 1 exit status
+configure:16446: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "encfs"
+| #define VERSION "1.7.4"
+| #define ENABLE_NLS 1
+| #define HAVE_GETTEXT 1
+| #define HAVE_DCGETTEXT 1
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_DLFCN_H 1
+| #define LT_OBJDIR ".libs/"
+| /* end confdefs.h.  */
+| #include <pthread.h>
+| int
+| main ()
+| {
+| pthread_t th; pthread_join(th, 0);
+|                      pthread_attr_init(0); pthread_cleanup_push(0, 0);
+|                      pthread_create(0,0,0,0); pthread_cleanup_pop(0);
+|   ;
+|   return 0;
+| }
+configure:16455: result: no
+configure:16363: checking whether pthreads work with -pthread
+configure:16446: gcc -o conftest -Wall -g -O2 -pthread   conftest.c   >&5
+conftest.c: In function 'main':
+conftest.c:31:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 1) [-Wnonnull]
+conftest.c:32:22: warning: null argument where non-null required (argument 3) [-Wnonnull]
+conftest.c:30:27: warning: 'th' is used uninitialized in this function [-Wuninitialized]
+configure:16446: $? = 0
+configure:16455: result: yes
+configure:16475: checking for joinable pthread attribute
+configure:16488: gcc -o conftest -Wall -g -O2 -pthread   conftest.c   >&5
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_build=x86_64-pc-linux-gnu
+ac_cv_c_compiler_gnu=yes
+ac_cv_cxx_compiler_gnu=yes
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-Wall -g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=set
+ac_cv_env_CXXFLAGS_value='-g -O2'
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_OPENSSL_CFLAGS_set=
+ac_cv_env_OPENSSL_CFLAGS_value=
+ac_cv_env_OPENSSL_LIBS_set=
+ac_cv_env_OPENSSL_LIBS_value=
+ac_cv_env_PKG_CONFIG_LIBDIR_set=
+ac_cv_env_PKG_CONFIG_LIBDIR_value=
+ac_cv_env_PKG_CONFIG_PATH_set=
+ac_cv_env_PKG_CONFIG_PATH_value=
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_RLOG_CFLAGS_set=
+ac_cv_env_RLOG_CFLAGS_value=
+ac_cv_env_RLOG_LIBS_set=
+ac_cv_env_RLOG_LIBS_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=x86_64-linux-gnu
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_header_dlfcn_h=yes
+ac_cv_header_inttypes_h=yes
+ac_cv_header_memory_h=yes
+ac_cv_header_stdc=yes
+ac_cv_header_stdint_h=yes
+ac_cv_header_stdlib_h=yes
+ac_cv_header_string_h=yes
+ac_cv_header_strings_h=yes
+ac_cv_header_sys_stat_h=yes
+ac_cv_header_sys_types_h=yes
+ac_cv_header_unistd_h=yes
+ac_cv_host=x86_64-pc-linux-gnu
+ac_cv_objext=o
+ac_cv_path_EGREP='/bin/grep -E'
+ac_cv_path_FGREP='/bin/grep -F'
+ac_cv_path_GMSGFMT=/usr/bin/msgfmt
+ac_cv_path_GREP=/bin/grep
+ac_cv_path_MSGFMT=/usr/bin/msgfmt
+ac_cv_path_MSGMERGE=/usr/bin/msgmerge
+ac_cv_path_SED=/bin/sed
+ac_cv_path_XGETTEXT=/usr/bin/xgettext
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_path_mkdir=/bin/mkdir
+ac_cv_prog_AWK=gawk
+ac_cv_prog_CPP='gcc -E'
+ac_cv_prog_CXXCPP='g++ -E'
+ac_cv_prog_ac_ct_AR=ar
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_ac_ct_CXX=g++
+ac_cv_prog_ac_ct_OBJDUMP=objdump
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_ac_ct_STRIP=strip
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_g=yes
+ac_cv_prog_cxx_g=yes
+ac_cv_prog_make_make_set=yes
+acl_cv_hardcode_direct=no
+acl_cv_hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+acl_cv_hardcode_libdir_separator=
+acl_cv_hardcode_minus_L=no
+acl_cv_libext=a
+acl_cv_libname_spec='lib$name'
+acl_cv_library_names_spec='$libname$shrext'
+acl_cv_path_LD=/usr/bin/ld
+acl_cv_prog_gnu_ld=yes
+acl_cv_rpath=done
+acl_cv_shlibext=so
+acl_cv_wl=-Wl,
+am_cv_CC_dependencies_compiler_type=gcc3
+am_cv_CXX_dependencies_compiler_type=gcc3
+gt_cv_func_CFLocaleCopyCurrent=no
+gt_cv_func_CFPreferencesCopyAppValue=no
+gt_cv_func_gnugettext1_libc=yes
+lt_cv_archive_cmds_need_lc=no
+lt_cv_deplibs_check_method=pass_all
+lt_cv_file_magic_cmd='$MAGIC_CMD'
+lt_cv_file_magic_test_file=
+lt_cv_ld_reload_flag=-r
+lt_cv_nm_interface='BSD nm'
+lt_cv_objdir=.libs
+lt_cv_path_LD=/usr/bin/ld
+lt_cv_path_LDCXX='/usr/bin/ld -m elf_x86_64'
+lt_cv_path_NM='/usr/bin/nm -B'
+lt_cv_prog_compiler_c_o=yes
+lt_cv_prog_compiler_c_o_CXX=yes
+lt_cv_prog_compiler_pic_works=yes
+lt_cv_prog_compiler_pic_works_CXX=yes
+lt_cv_prog_compiler_rtti_exceptions=no
+lt_cv_prog_compiler_static_works=yes
+lt_cv_prog_compiler_static_works_CXX=yes
+lt_cv_prog_gnu_ld=yes
+lt_cv_prog_gnu_ldcxx=yes
+lt_cv_shlibpath_overrides_runpath=no
+lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[	 ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[	 ][	 ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p'\'''
+lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \([^ ]*\) $/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \([^ ]*\) $/  {\"\1\", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \(lib[^ ]*\)$/  {"\2", (void *) \&\2},/p'\'' -e '\''s/^[ABCDGIRSTW]* \([^ ]*\) \([^ ]*\)$/  {"lib\2", (void *) \&\2},/p'\'''
+lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[ABCDGIRSTW]* .* \(.*\)$/extern char \1;/p'\'''
+lt_cv_sys_max_cmd_len=3458764513820540925
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run aclocal-1.11'
+AMDEPBACKSLASH='\'
+AMDEP_FALSE='#'
+AMDEP_TRUE=''
+AMTAR='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run tar'
+AR='ar'
+AUTOCONF='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run automake-1.11'
+AWK='gawk'
+BOOST_CPPFLAGS=''
+BOOST_FILESYSTEM_LIB=''
+BOOST_LDFLAGS=''
+BOOST_SERIALIZATION_LIB=''
+BOOST_SYSTEM_LIB=''
+BUILD_MANHTML_FALSE=''
+BUILD_MANHTML_TRUE=''
+BUILD_MAN_FALSE=''
+BUILD_MAN_TRUE=''
+BUILD_NLS_FALSE=''
+BUILD_NLS_TRUE=''
+BUILD_OPENSSL_FALSE=''
+BUILD_OPENSSL_TRUE=''
+BUILD_SSLCIPHER_FALSE=''
+BUILD_SSLCIPHER_TRUE=''
+BUILD_STATIC_FALSE=''
+BUILD_STATIC_TRUE='#'
+CC='gcc'
+CCDEPMODE='depmode=gcc3'
+CFLAGS='-Wall -g -O2 -pthread'
+CPP='gcc -E'
+CPPFLAGS=''
+CXX='g++'
+CXXCPP='g++ -E'
+CXXDEPMODE='depmode=gcc3'
+CXXFLAGS='-g -O2'
+CYGPATH_W='echo'
+DARWIN_FALSE=''
+DARWIN_TRUE='#'
+DEFS=''
+DEPDIR='.deps'
+DSYMUTIL=''
+DUMPBIN=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP='/bin/grep -E'
+EXEEXT=''
+FGREP='/bin/grep -F'
+FUSE_LIBS=''
+GETTEXT_MACRO_VERSION='0.17'
+GMSGFMT='/usr/bin/msgfmt'
+GMSGFMT_015='/usr/bin/msgfmt'
+GREP='/bin/grep'
+HAVE_EVP_AES=''
+HAVE_EVP_BF=''
+HAVE_HMAC_INIT_EX=''
+HAVE_SSL=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
+INTLLIBS=''
+INTL_MACOSX_LIBS=''
+LD='/usr/bin/ld -m elf_x86_64'
+LDFLAGS=''
+LIBICONV='-liconv'
+LIBINTL=''
+LIBOBJS=''
+LIBS=' '
+LIBTOOL='$(SHELL) $(top_builddir)/libtool'
+LIPO=''
+LN_S='ln -s'
+LTLIBICONV='-liconv'
+LTLIBINTL=''
+LTLIBOBJS=''
+MAKEINFO='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/missing --run makeinfo'
+MKDIR_P='/bin/mkdir -p'
+MSGFMT='/usr/bin/msgfmt'
+MSGFMT_015='/usr/bin/msgfmt'
+MSGMERGE='/usr/bin/msgmerge'
+NM='/usr/bin/nm -B'
+NMEDIT=''
+OBJDUMP='objdump'
+OBJEXT='o'
+OPENSSL_CFLAGS=''
+OPENSSL_LIBS=''
+OTOOL64=''
+OTOOL=''
+PACKAGE='encfs'
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME=''
+PACKAGE_STRING=''
+PACKAGE_TARNAME=''
+PACKAGE_URL=''
+PACKAGE_VERSION=''
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+PKG_CONFIG_LIBDIR=''
+PKG_CONFIG_PATH=''
+POD2HTML=''
+POD2MAN=''
+POSUB='po'
+PTHREAD_CC=''
+PTHREAD_CFLAGS='-pthread'
+PTHREAD_LIBS=''
+RANLIB='ranlib'
+RLOG_CFLAGS=''
+RLOG_LIBS=''
+SED='/bin/sed'
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP='strip'
+USE_NLS='yes'
+VERSION='1.7.4'
+XGETTEXT='/usr/bin/xgettext'
+XGETTEXT_015='/usr/bin/xgettext'
+XGETTEXT_EXTRA_OPTIONS=''
+ac_ct_CC='gcc'
+ac_ct_CXX='g++'
+ac_ct_DUMPBIN=''
+acx_pthread_config=''
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE='#'
+am__fastdepCC_TRUE=''
+am__fastdepCXX_FALSE='#'
+am__fastdepCXX_TRUE=''
+am__include='include'
+am__isrc=''
+am__leading_dot='.'
+am__quote=''
+am__tar='${AMTAR} chof - "$$tardir"'
+am__untar='${AMTAR} xf -'
+bindir='${exec_prefix}/bin'
+build='x86_64-pc-linux-gnu'
+build_alias='x86_64-linux-gnu'
+build_cpu='x86_64'
+build_os='linux-gnu'
+build_vendor='pc'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host='x86_64-pc-linux-gnu'
+host_alias=''
+host_cpu='x86_64'
+host_os='linux-gnu'
+host_vendor='pc'
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='${SHELL} /home/steve/Packages/NMU/encfs-1.7.4/install-sh'
+libdir='${exec_prefix}/lib'
+libexecdir='${exec_prefix}/libexec'
+localedir='${datarootdir}/locale'
+localstatedir='${prefix}/var'
+mandir='${prefix}/share/man'
+mkdir_p='/bin/mkdir -p'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='${prefix}/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME ""
+#define PACKAGE_TARNAME ""
+#define PACKAGE_VERSION ""
+#define PACKAGE_STRING ""
+#define PACKAGE_BUGREPORT ""
+#define PACKAGE_URL ""
+#define PACKAGE "encfs"
+#define VERSION "1.7.4"
+#define ENABLE_NLS 1
+#define HAVE_GETTEXT 1
+#define HAVE_DCGETTEXT 1
+#define STDC_HEADERS 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_STRINGS_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_STDINT_H 1
+#define HAVE_UNISTD_H 1
+#define HAVE_DLFCN_H 1
+#define LT_OBJDIR ".libs/"
+
+configure: caught signal 2
+configure: exit 1
